dev/.env.example

```diff
@@ -12,3 +12,8 @@ MYSQL_ROOT_PASSWORD=SET_A_PASSWORD_HERE_123
 # Complete using the values from the Manage SSO page in the web vault
 IDP_SP_ENTITY_ID=http://localhost:51822/saml2
 IDP_SP_ACS_URL=http://localhost:51822/saml2/yourOrgIdHere/Acs
+
+# Optional reverse proxy configuration
+# Should match server listen ports in reverse-proxy.conf
+API_PROXY_PORT=4100
+IDENTITY_PROXY_PORT=34656
```

Comment:
Sort of a nitpick and maybe there is a good reason for it not doing this but can we make the identity port be `33756` so that both ports are `n + 100`?

Reply:
That shouldn't be a problem, I was only just incrementing the second digit by 1. But if `n + 100` makes more sense that's good with me!

Reply:
We have a similar setup for self-host versions of cloud services doing `n + 1`. And in here you have `n + 2` for a second instance of a single service. Even though second digit incremented by one IS a predictable pattern it seems easier to standardize around it being `n + 100`.